.github/workflows/elasticsearch_online.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         restore-keys: |
           gt-maven-
     - name: Build GeoTools dependent modules (no tests, prepare fresh artifacts)
-      run: mvn -B clean install -T1C -Dall --file pom.xml -DskipTests -pl modules/unsupported/elasticsearch -am -Dspotless.apply.skip=true
+      run: mvn -B clean install -T1C -Dall --file pom.xml -DskipTests -pl platform-dependencies,bom,modules/unsupported/elasticsearch -am -Dspotless.apply.skip=true
```

Comment:
This is going to be a bit of a pain, one has to remember adding the two profiles when the bom is modified and one wants to build a specific module, right?

Is this documented anywhere? Haven't seen it, but might have just overlooked it.

Reply:
yes it worked fine, thanks.